lib/xlsx/xform/sheet/data-validations-xform.js

```diff
@@ -28,17 +28,65 @@ function assignBool(definedName, attributes, name, defaultValue) {
   }
 }
 
+function mergeDataValidations(model) {
```

Comment:
Could you add a comment describing how the model is being optimised here?

Reply:
currently excelJS model stores dataValidation per each cell. So if there is many cells data validation is created for every cell. I'll give you example: on my project there 5k rows x 20 columns ~ 100k cells. So in this case MS Excel openes generated file very long (minutes) or if some number is exceeded it reports that ms excel is corrupted.
In many cases validation is the same for bigger range (in our case it is while column but header). So my optimization just replaces multiple dataValidation xml nodes(with the same formula and continous range) with with one where address is defined as range instead one cell. So on my project instead 100k dataValidation nodes there is 20 and this file is opened by MS Excel fast and without errors.